webpack_loader/utils.py

```diff
@@ -124,3 +124,10 @@ def get_static(asset_name, config='DEFAULT'):
         public_path = getattr(settings, 'STATIC_URL')
 
     return '{0}{1}'.format(public_path, asset_name)
+
+def get_asset(asset_name, config='DEFAULT'):
+    loader = get_loader(config)
+    asset = loader.get_asset(asset_name)
+    if not asset:
+        return get_static(asset_name, config)
+    return get_static(asset['name'], config)
```

Comment:
I guess this should not support the old `asset_name` based lookup to avoid confusion.